test/integration/api.bats

```diff
@@ -352,9 +352,27 @@ function teardown() {
 	skip
 }
 
-# FIXME
 @test "docker rename" {
-	skip
+	start_docker 3
```

Comment:
Could you check conflict detection in the integration test?

You could create a container named `test-1` on node-1 and a container named `test-2` on node-2.

Then try to rename `test-2` into `test-1` and check that it fails.

Then try to rename `test-2` into `test-3` and check that it succeeds.